docs/operation/account_administration.rst

```diff
@@ -8,61 +8,35 @@ changing the default username and password for some MetalK8s services.
 
 .. _ops-grafana-admin:
 
-Administering Grafana
-*********************
+Administering Grafana and MetalK8s GUI
```

Comment:
I think this doc needs to be rewritten, to show the following sections:

- Dex and OIDC, how it works (connectors, static users...)
- Services relying on Dex (K8s API, Salt API, Grafana, MetalK8s GUI...)
- Procedures to add/edit static users
- Procedures to configure connectors (maybe not filled in right now)

Or, if we really want to put everything under a single "Cluster and Services Configurations" document (I'd vote against this, harder to discover), then maybe just remove this document entirely.

Reply:
Done!